lib/common.sh

```diff
@@ -134,6 +134,21 @@ is_ip_port_reachable()
     nc -w 3 -z $ip $port > /dev/null 2>&1
 }
 
+#
+# Verify if FQDN is resolved into ipv4_addr by /etc/hosts entry.
+#
+is_present_in_etc_hosts() {
+    local ip="$1"
+    local hostname="$2"
+
+    # Search for the entry in /etc/hosts
+    if grep -qE "^[[:space:]]*$ip[[:space:]]+[^#]*\<$hostname\>" /etc/hosts; then
+        return 0
+    fi
+
+    return 1
```

Comment:
nit,
you can simply replace this with
grep -qE "^[[:space:]]*$ip[[:space:]]+[^#]*\<$hostname\>" /etc/hosts